### DISCUSSION TITLE
Thank you all... pre-orders, one year on...

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#1015] [General]
One year ago today, the pre-order offer for twinBASIC was launched.  Doesn't time fly when you're having such fun?!  Whilst we're not at the v1 stage yet, we are getting closer day by day.

From the bottom of my heart, I wish to thank each and every in one of you for your amazing support, whether that be financially, technically, or both.  Without you guys, this project just could not exist.

100 GitHub stars... 1000 GitHub issues/discussions... approaching 100 discord members... around 50 downloads per day... interest in tB is increasing all the time, and it's a pleasure to see new names and faces around.  I am immensely proud of what we have achieved together so far, and am very much looking forward to the next 12 months!

For those of you that joined us at the start, and paid annually, the subscription fees will be coming due for renewal, so please look out for an email from us.  If you paid by PayPal, remember that the subscription payments will be taken automatically unless you've told them otherwise.  

I know times are hard for many at the moment, so if there are any problems please just get in touch.

_ps. for those of you following on discord the other day... just in case you're wondering... I don't really drive a Lamborghini 🤣... every penny that comes in helps keep this project alive._

**Comment by rexxitall** [comment]
Wayne a Lambo is a shitty car for posers. :) Did you know that 70% all ever made Porsche are still in use ? So Porsche is way more reliable :) *grin* And way more affordable :) After 1984 all are also ZN plated and do not rust :) Thanks for your great work and patience. - And please buy a Porsche :)

**Comment by mburns08109** [comment]
> I don't really drive a Lamborghini 🤣

Then why was I preparing to buy you tires for it!??  O_O

**Reply by WaynePhillipsEA** [reply]
Just in preparation 🤞

**Comment by saturday220514** [comment]
If even Fiat Puntos got the love, care and easy life that Porsches do, 70% of them might still still be in use too.
